lms/envs/common.py

```diff
@@ -1795,7 +1795,7 @@
     'certificates',
     'dashboard',
     'instructor',
-    'instructor_task',
+    'lms.djangoapps.instructor_task',
```

Comment:
that doesn't make sense to me — what if another one of these apps has a migration?